example/example-node.js

```diff
@@ -23,6 +23,7 @@ const htmlString = `<!DOCTYPE html>
                 src="https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png"
                 alt="Red dot"
             />
+            <blockquote>Quote <span>from <a href="https://www.wikipedia.org/">Wikipedia</a></span></blockquote>
```

Comment:
See other comment, unable to see this in the document and not seeing it render in the document.xml